x/aol/keeper/owner_test.go

```diff
@@ -0,0 +1,95 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/medibloc/panacea-core/types/testsuite"
+	aoltypes "github.com/medibloc/panacea-core/x/aol/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+	"testing"
+)
+
+var (
+	pubKeys = []crypto.PubKey{
+		secp256k1.GenPrivKey().PubKey(),
+		secp256k1.GenPrivKey().PubKey(),
+	}
+
+	addresses = []sdk.AccAddress{
+		sdk.AccAddress(pubKeys[0].Address()),
+		sdk.AccAddress(pubKeys[1].Address()),
+	}
+)
+
+type ownerTestSuite struct {
+	testsuite.TestSuite
+}
+
+func TestOwnerKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(ownerTestSuite))
+}
+
+func (suite ownerTestSuite) TestOneOwner() {
+	ctx := suite.Ctx
+	aolKeeper := suite.AolKeeper
+
+	key := aoltypes.OwnerCompositeKey{
+		OwnerAddress: addresses[0],
+	}
+	owner := aoltypes.Owner{
+		TotalTopics: 1,
+	}
+	aolKeeper.SetOwner(ctx, key, owner)
+
+	// verify HasOwner
+	suite.Require().True(aolKeeper.HasOwner(ctx, key))
+
+	// verify GetOwner
+	resultOwner := aolKeeper.GetOwner(ctx, key)
+	suite.Require().Equal(owner, resultOwner)
+
+	// verify GetAllOwner
+	resultKeys, resultOwners := aolKeeper.GetAllOwners(ctx)
+	suite.Require().Equal(1, len(resultKeys))
+	suite.Require().Equal([]aoltypes.OwnerCompositeKey{key}, resultKeys)
+	suite.Require().Equal(1, len(resultOwners))
+	suite.Require().Equal([]aoltypes.Owner{owner}, resultOwners)
+}
+
+func (suite ownerTestSuite) TestMultiOneOwner() {
```

Comment:
If I understand it correctly, I think we can name this test as `TestMultiOwners`.

Reply:
That's right! I changed method name